_sass/modules/_nav.scss

```diff
@@ -1,105 +1,289 @@
-// Top Static Nav
-#common-nav {
-  background-color: transparent;
-  padding-top: $spacing--xs;
-
-  .nav {
-    display: none;
-    font-size: 13px;
-    letter-spacing: 1px;
-    text-align: right;
-    position: relative;
-    z-index: 9012;
+//Navigation
+.navigation-opened {
+  @media(max-width: $tablet){
+    overflow: hidden;
```

Comment:
Improve behavior for open/close  states. 
We have wrong position of  `close` icon in `navigation-opened` state.  Because of hidden scroll and jumping content by same reason. Add please `17px` compensation.    

![image](https://user-images.githubusercontent.com/17984255/47089319-40f39f00-d229-11e8-99aa-707de61a6fcd.png)

Reply:
It is not necessary for mobile devices. 
We can add this only for the desktop devices for the Windows.

Reply:
Got it, thanks.  Let's leave it on this stage.